### DISCUSSION TITLE
API Authentication

### DISCUSSION THREAD

**Discussion by Stelikas** [Kyon147/laravel-shopify#57] [Q&A] [answered]
Looking at the Wiki API Authentication is still in progress, i would like to extend Shopify API and create an endpoint to retrieve product by it's metafield, although when i try to do an API call "No authenticated user or shop domain in file", how do i make authenticated API calls through the API?

**Comment by Kyon147** [comment, marked as answer]
There is detail on this page to get you started https://github.com/Kyon147/laravel-shopify/wiki/Usage#accessing-api-for-the-current-shop

You need to load up the shop model you wish to access, then can use `->api()`
